maps/biter_battles_v2/gui.lua

```diff
@@ -386,6 +429,23 @@ function Public.create_statistics_gui_button(player)
     frame.visible = false
 end
 
+local function add_feed_button(element, name, tooltip)
+    local f = element.add({
+        type = 'sprite-button',
+        name = name,
+        sprite = 'item/' .. name,
+        style = 'slot_button',
+        tooltip = tooltip,
+    })
+    gui_style(f, { padding = 0 })
```

Comment:
I know it's not the main focus of the PR, but for simple (single-parameter) styles, it might be better to just call `.style`
```suggestion
    element.add({
        type = 'sprite-button',
        name = name,
        sprite = 'item/' .. name,
        style = 'slot_button',
        tooltip = tooltip,
    }).style.padding = 0
```